src/core/server/index.ts

```diff
@@ -86,6 +86,7 @@ export {
   IScopedClusterClient,
   ElasticsearchConfig,
   ElasticsearchClientConfig,
+  ElasticsearchConfigType,
```

Comment:
@restrry Is there another way around doing this? I don't like having to make a change here

Reply:
```typescript
import { config } from 'src/core/server';
import { TypeOf } from '@kbn/config-schema';
type ElasticsearchConfigType = TypeOf<typeof cfg.elasticsearch.schema>;
```

Reply:
Thanks!